java/BUILD

```diff
@@ -16,3 +16,21 @@ docker_build(
         # cmd = ["/path/to/deploy.jar", "--option1", ...]
     ],
 )
+
+# The default jetty tarball has a top-level versioned directory. We need to strip this directory
+# so that we can unpack directly into /jetty in our final image.
+genrule(
+    name = "unpacked_jetty",
+    srcs = ["@jetty//file"],
+    outs = ["_unpacked_jetty.tar.gz"],
+    cmd = "mkdir -p $(@D)/jetty && tar --strip-components=1 -xzf $(<) -C $(@D)/jetty && tar -czf $(@) -C $(@D) jetty  && rm -rf $(@D)/jetty",
+    visibility = ["//visibility:private"],
+)
+
+docker_build(
+    name = "jetty",
+    base = ":java8",
+    cmd = ["/jetty/start.jar"],
+    tars = [":_unpacked_jetty.tar.gz"],
+    workdir = "/jetty/demo-base/",
```

Comment:
I think workdir should be `/jetty`, so that users don't have to override it.  Can you add a blurb similar to what we have for Java about how we expect folks to consume this?  e.g. Add your WAR under `/jetty/webapps`